### PR TITLE
Revert "test coverage lib/debug"

### PR DESCRIPTION
**Pull request by Tomasz69**
Reverts OpenZWave/Zwave2Mqtt#390

### CODE REVIEW

**Comment by robertsLando** [comment]
@Tomasz69 Has it broke something? Everything was working on my tests

**Comment by robertsLando** [comment]
@chrisns Do you know?

**Comment by chrisns** [comment]
@robertsLando no, i'd resisted responding with a "works for me" :)

**Comment by robertsLando** [comment]
Let's see it @Tomasz69 tell us what it has break

File: lib/debug.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+var debug = require('debug')
+
+if (!process.env.DEBUG) {
+  debug.enable('z2m:*')
 }
-init()
+
+debug = debug('z2m')
+
+debug.log = console.log.bind(console)
```
Comment (robertsLando) [inline lib/debug.js:9]: Without this debug outputs to stderr @chrisns . You should add back this line. When using tools like pm2 everything will go to error log file.

**Comment by robertsLando** [comment]
@chrisns I suggest to open a new PR with that fix in the review, just add that line

**Pull request closed without merging**